### DISCUSSION TITLE
Display whether a plugin depends on AngularJS

### DISCUSSION THREAD

**Discussion by sympatheticmoose** [grafana/grafana#65605] [Plugins]
To help users understand how they may be impacted by the deprecation of Angular initially announced before Grafana v9, and to encourage migrations prior to this support being disabled by default and ultimately removed. We should highlight components which have a dependency on Angular more prominently within the Grafana UI. 

This could include:
- A warning banner when we detect a loaded angular plugin within the Grafana instance
- A warning when the user navigate to a dashboard which has an Angular dependency (could we even list the exact component i.e. a panel?)
- An `Angular` warning label visible in the plugin catalog (this could be `<user>.grafana.net/plugins` similar to Core/Installed, and/or the Signed badge within a given plugins details page)
- Similar for the connections page `<user>.grafana.net/connections/connect-data`

The user will need to know what to do with this warning and the suggestion is to link initially to the angular support deprecation notice: https://grafana.com/docs/grafana/latest/developers/angular_deprecation/

Things to consider:
- how we 'detect' angular - for the catalog we have a known list of plugins, but that would omit any private plugins in OSS or on-premise Enterprise
- severity of the alert
- whether/how a user is able to dismiss the notification
  - how they could rediscover the warning having dismissed previously

**Comment by gelicia** [comment]
I believe this is done, but feel free to open an issue if it is not.
